tests/lambda_functions/vetext_incoming_forwarder_lambda/test_vetext_incoming_forwarder_lambda.py

```diff
@@ -413,12 +448,12 @@ def test_loading_message_from_alb_fails(mocker, monkeypatch, all_path_env_param_
     mocker.patch('base64.b64decode', side_effect=base64.binascii.Error)
     response = vetext_incoming_forwarder_lambda_handler(albInvokedWithoutAddOn, False)
 
-    assert response['statusCode'] == 500
+    assert response['statusCode'] == 403
     assert response['body'] == '<Response />'
     assert response['headers'] is not None
     assert response['headers']['Content-Type'] == 'text/xml'
 
-    sqs_dead_letter_mock.assert_called_once()
+    sqs_dead_letter_mock.assert_not_called()
```

Comment:
An un-decodable message results in a 403, and no retry is currently performed. Part of the ticket that will be created (see comment above).